### DISCUSSION TITLE
Download dashboard details

### DISCUSSION THREAD

**Discussion by kerenaba** [langfuse/langfuse#917] [Ideas]
Hi,
I'm Keren Abadi from Amdocs.
We must have the ability to download from langfuse the results including the feedbacks.
It’s the only way for us to investigate & improve the tool.
 - User ID
 - Timestemp
 - Score
 - Feedback

**Comment by j2deen** [comment]
This would be useful as sometimes we need to present reports on improvements into other tools and it would make it much easier to do so - thanks.

**Comment by khareyash05** [comment]
Hey @kerenaba If I understood correctly, an option to download the dashboard is what you are highlighting right?

**Comment by marcklingen** [comment]
The data is fully available via the API, is that sufficient for your downstream use case @kerenaba?
Reference: https://langfuse.com/docs/api-reference

**Comment by marcklingen** [comment]
You can now download the metrics from custom charts in dashboards in the UI: https://langfuse.com/changelog/2025-06-17-csv-download-dashboard-charts
